docs/zh/README.md

```diff
@@ -28,6 +28,9 @@ user:
 news:
   title: 事件 & 新闻
   content:
+  - header: 欢迎华卫成为新的committer
```

Comment:
Remove the oldest one to only keep 4 pieces of news, the same as the English version

Reply:
ok